config.json

```diff
@@ -95,19 +52,18 @@
       ]
     },
     {
-      "slug": "simple-cipher",
-      "uuid": "62d60b42-93bc-4de9-90d1-1ca18a847812",
+      "slug": "bob",
```

Comment:
I'm wondering if `bob` is actually a difficulty 2 exercise. I've seen so many people struggle with it in other tracks, that I'm now moving it farther to the end of the track. But maybe it is differently in JavaScript.

Reply:
It's not. Regex is medium. We probably want more exercises in-between